src/main/java/uk/gov/hmcts/reform/sendletter/logging/AppInsights.java

```diff
@@ -52,77 +51,6 @@ public class AppInsights {
     public AppInsights() {
 
     }
-    // schedules
```

Comment:
This creates a Request log that just says Schedule /UploadLettersTask and overwrites the application insights json. Doesn't seem to give valuable info